docs/OMS-Agent-for-Linux.md

```diff
@@ -181,8 +182,8 @@ sudo ./omsadmin.sh -w <WorkspaceID> -s <Shared Key>
 WORKSPACE_ID=<WorkspaceID>
 SHARED_KEY=<Shared Key>
 ```
-3.	Restart the omsagent:
-`sudo service omsagent restart`
+3.	Register the omsagent service and restart:
+`sudo ./service_control enable`

```

Comment:
I think this is wrong. You still want to use the restart command.

The restart command will register the service, enable the service (or it should upon registration), and then start the service. This is what you want.

If you simply enable it, that will enable restarting the service at reboot time, which is a subset of what you want.

Another iteration please, or just throw this away since existing code should be fine. If not, then another iteration please.